packages/build-tool/README.md

```diff
@@ -90,6 +123,34 @@ module.exports = {
 ```js
 import defaultConfig from '@alleyinteractive/build-tool/dist/esm/config/webpack.config';
 ```
+#### Using Webpack `extends` property
+
+Similar to the spread operator, you can also use the `extends` property in the `webpack.config.js` file to extend the default configuration.
+> The extends property allows you to extend an existing configuration to use as the base. It internally uses the webpack-merge package to merge the configurations and helps you to avoid duplicating configurations between multiple configurations.
```

Comment:
```suggestion
> The extends property allows you to extend an existing configuration to use as the base. It internally uses the webpack-merge package to merge the configurations and helps you to avoid duplicating configurations.
```